docs/source/HTF.rst

```diff
@@ -0,0 +1,54 @@
+
+Key findings for the CAPE case are listed below:
+
+- The lower CAPE in GFS is attributed to air at low levels being drier than observed.
+- The larger-than-observed Bowen ratio in GFS suggests an incorrect surface energy partitioning associated with soil moisture and soil temperature. 
+- GFS v16 exhibits more vertical mixing leading to a deeper boundary layer than v15.2 and observations.
+- The drier air in GFSv16 is mostly related to local physics not large-scale advection in this case.
+- With realistic IC of vertical profiles and surface fluxes, the GFS v16 physics suite is able to produce the large CAPE.
+
+..............................
+2020 Cold Air Damming
+..............................
+
+We adopted the latest version of Short-Range Weather (SRWeather) App (version 1.0) to investigate the 2020 CAD case1 developed as part of the UFS Case Studies project to investigate the impact of code updates, horizontal and vertical resolution, and physics suites. The SRWeather App is the foundation for building NOAA’s future convection-allowing Rapid Refresh Forecast System (RRFS). The limited area configuration needs fewer computer resources and can run models in high resolution and cycle frequently. Thus the case studies conducted with the SRWeather App are a useful supplement for the global configuration case studies. The physics suites used in this study, which focuses on the CONUS domain, include GFS v15.2, GFS v16, GFS v17α, and RRFS v1α. The results from UFS SRWeather App were compared with the results from the UFS global configuration. The SRWeather App was also tested with different configurations, such as different versions of the code base, model grid spacing (13 km vs 3 km, 64 levels vs 127 levels), and physics suite employed. The model forecasts were verified against station observations and analysis data. 
+
+The key findings for the CAD case are:
+
+- The spatial pattern of the 2-meter surface temperature field obtained with the SRWeather App is slightly improved over the one obtained with the global configuration.
+- Using the code version of April 10 th, 2022 and more vertical levels leads to
+- Improvements for this case when using the GFS v16 suite
+- Degradation when using the (uncoupled) GFS v17α suite
+- When the 3-km horizontal grid spacing configuration is used with the GFS v15.2 and GFS v16 suites (but not with the GFS v17α suite), the cold air is not maintained and the CAD strength is reduced, which is a degradation when compared to experiments conducted at lower resolutions. for
+- Sensitivity tests with suite GFS v17α show that changing the land surface model from NoahMP to Noah can cause a northward shift of the cold air.
+
+**References**
```

Comment:
Can you add links to the references?

Reply:
@ligiabernardet Links added.

Reply:
Suggestion: 
Sensitivity tests with suite GFS v17α show that changing the land surface model from NoahMP to Noah can cause a northward shift of the cold air, which leads to a warmer 2-m temperature bias and a weaker CAD signal.